pkg/scalers/selenium_grid_scaler.go

```diff
@@ -238,5 +243,11 @@ func getCountFromSeleniumResponse(b []byte, browserName string, browserVersion s
 		}
 	}
 
+	var gridMaxSession = int64(seleniumResponse.Data.Grid.MaxSession)
+
+	if gridMaxSession > 0 {
+		count = (count + gridMaxSession - 1) / gridMaxSession
```

Comment:
Isn't enough the division? I mean, shouldn't be the response only `count / gridMaxSession`?

Reply:
Not always. Imagine we have 1 pending session and 10 `maxSession` setting value. We will need at least one node to process the request, but since `1/10=0` we will be waiting until 9 others sessions be queued to scale to 1.